src/irclog2html/irclog2html.py

```diff
@@ -875,6 +875,12 @@ def parse_args(argv=sys.argv):
     parser.add_option('-o', '--output-file',
                       help="destination output file or directory"
                            " (default: <input-file-name>.html)")
+    parser.add_option('--hide-event', dest="hide_events", action='append',
+                      choices=['COMMENT', 'ACTION', 'JOIN', 'PART',
+                               'NICKCHANGE', 'SERVER', 'OTHER'], default=[],
+                      help="Event to ignore and not add to the output. Can be"
+                           "one of COMMENT, ACTION, JOIN, PART, NICKCHANGE,"
+                           "SERVER, OTHER. Can be given multiple times.")
```

Comment:
If you look at the output of `--help`, you should notice that someword are joinedtogether.

Personally, I've decided on a rule that when splitting text across two lines, I'll put the whitespace on the next line, so it's immediately noticeable when it's missing.

```suggestion
                      help="Event to ignore and not add to the output. Can be"
                           " one of COMMENT, ACTION, JOIN, PART, NICKCHANGE,"
                           " SERVER, OTHER. Can be given multiple times.")
```